packages/core/src/modules/credentials/protocol/v2/__tests__/v2.ldproof.credentials.propose-offerED25519.test.ts

```diff
@@ -325,30 +319,30 @@ describe('credentials', () => {
       messageClass: V2OfferCredentialMessage,
     })
 
-    const credOfferJson = offerMessage?.offerAttachments[1].getDataAsJson<SignCredentialOptionsRFC0593>()
+    const credOfferJson = offerMessage?.offerAttachments[1].getDataAsJson<JsonLdSignCredentialFormat>()
 
     expect(credOfferJson).toMatchObject({
       credential: {
-        context: [
+        '@context': [
           'https://www.w3.org/2018/credentials/v1',
           'https://w3id.org/citizenship/v1',
           'https://w3id.org/security/bbs/v1',
         ],
         id: 'https://issuer.oidp.uscis.gov/credentials/83627465',
         type: ['VerifiableCredential', 'PermanentResidentCard'],
         issuer: 'did:key:z6Mkgg342Ycpuk263R9d8Aq6MUaxPn1DDeHyGo38EefXmgDL',
-        identifier: '83627465',
-        name: 'Permanent Resident Card',
```

Comment:
Any reason to move these to the credentialSubject? The example has them top level. Does TS give errors in this case? We should resolved that then